src/qt/res/stylesheets/dark_stylesheet.qss

```diff
@@ -370,20 +370,17 @@ QCheckBox{
     padding-top:1.25em;
     min-width:8em;
     height:100%;    
-    background: transparent;
     color: white;
 }
 
 #toolbar2 {
     border:none;
-    background-color:qlineargradient(x1: 0, y1: 0, x2: 0.5, y2: 0.5,stop: 0 rgb(45,45,50), stop: 1 rgb(45,45,50));
```

Comment:
I imagine people will complain about this color changing on the light/dark themes for mainnet... don't think it's worth messing around with for now since we're expecting a new GUI in the future.

Reply:
I took a look of course, and I don't think it is bad...

Reply:
I don't think so either 🙂